PermissionScope/PermissionScope.swift

```diff
@@ -93,6 +98,17 @@ public class PermissionScope: UIViewController, CLLocationManagerDelegate, UIGes
     var authChangeClosure: ((Bool, [PermissionResult]) -> Void)? = nil
     var cancelClosure: (([PermissionResult]) -> Void)? = nil
 
+    // Computed variables
+    var allAuthorized: Bool {
```

Comment:
I didn't commit that part since using the variables would include an extra array loop (not that it matters since there are only 3 elements).
I'll commit those changes if you want (tomorrow probably)